apps/strapi-cms/src/components/common/cta.json

```diff
@@ -0,0 +1,27 @@
+{
+  "collectionName": "components_common_ctas",
+  "info": {
+    "displayName": "HeroCardCta",
```

Comment:
I would use a more generic CallToAction component (maybe using the Link component)